aip/general/0130.md

```diff
@@ -0,0 +1,88 @@
+| **Custom Fetch Methods**           |                                                                                                                                                                                                               |                                                    |                                 |                 |                |                 |
+| Custom collection fetch methods    | Fetch information across a collection that cannot be expressed via a standard method.                                                                                                                         | [AIP-136][]                                        | automatable                     | automatable     | automatable    | automatable     |
+| Custom resource fetch methods      | Fetch information for a single resource that cannot be expressed via a standard method.                                                                                                                       | [AIP-136][]                                        | automatable                     | automatable     | automatable    | automatable     |
+| **Custom Mutation Methods**        |                                                                                                                                                                                                               |                                                    |                                 |                 |                |                 |
+| Backing up a resource              | Storing a copy of a resource at a particular point in time.                                                                                                                                                   | [AIP-162][]                                        | unused or handwritten           | automatable     | automatable    | automatable     |
+| Restoring a resource               | Setting a resource to a version from a particular point in time.                                                                                                                                              | [AIP-162][]                                        | unused or handwritten           | automatable     | automatable    | automatable     |
+| Renaming a resource                | Modify the resource's name or id while preserving configuration and state.                                                                                                                                    | [AIP-136][]                                        | unused or handwritten           | automatable     | automatable    | automatable     |
```

Comment:
I suggest "data" rather than "state" since "state" has a particular meaning in AIP 216.

Reply:
ack, agreed.